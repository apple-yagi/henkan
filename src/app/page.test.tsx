import { render } from "@testing-library/react";
import { it, expect } from "vitest";

import Page from "./page";

it("should render the page", () => {
  const { asFragment, getByText } = render(<Page />);

  expect(getByText("Hello World")).toBeInTheDocument();
  expect(asFragment()).toMatchSnapshot();
});
