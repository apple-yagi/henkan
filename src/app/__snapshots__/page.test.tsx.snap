// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render the page 1`] = `
<DocumentFragment>
  <main
    class="flex min-h-screen flex-col items-center justify-between p-24"
  >
    <h1
      class="🐻-3154563503"
    >
      Hello World
    </h1>
  </main>
</DocumentFragment>
`;
